server/image_processor.py

```diff
@@ -0,0 +1,38 @@
+import sys
+
+from PIL import Image
+
+size = 35, 35
+angular_segments_count = 120
+angular_segments = 360 / angular_segments_count
+
+
+def process_image_for_mcu(image_file):
+    image = Image.open(image_file)
+    image.thumbnail(size)
+
+    output_pixels = []
+    for j in range(size[0]):
+        output_pixels.append([])
+
+    for theta in range(angular_segments_count):
+        copied_image = image.copy()
+        rotated_image = copied_image.rotate((360/angular_segments_count) * theta)
+        pixels = list(rotated_image.getdata())[(35 * 17):(35 * 17) + 35]
+        for i in range(len(pixels)):
+            output_pixels[i].append(pixels[i])
+
+    return output_pixels
+
+
+def print_pixels_as_mcu_code(rgb_data):
+    print('.ledValues = {\n')
+    for row in rgb_data:
+        crgbSrings = list(map(lambda pixel: f"CRGB({pixel[0]},{pixel[1]},{pixel[2]})", row))
+        print(f"{{{','.join(crgbSrings)}}},")
```

Comment:
Is this comma here at the end the bug you were talking about?

Reply:
Yeah this is the one. Can't believe I naively missed this. I fixed by adding a `join` instead of blindly concatenating each row with a comma.